1_train_and_save_LSTM.ipynb

```diff
@@ -664,7 +662,44 @@
+      "    resp = resource(*args, **kwargs)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/flask/views.py\", line 89, in view\n",
+      "    return self.dispatch_request(*args, **kwargs)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/flask_restful/__init__.py\", line 573, in dispatch_request\n",
+      "    resp = meth(*args, **kwargs)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/neuraxle/api/flask.py\", line 136, in get\n",
+      "    return wrapped.transform(request.get_json())\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/neuraxle/pipeline.py\", line 74, in transform\n",
+      "    data_container = self._transform_core(data_container, context)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/neuraxle/pipeline.py\", line 250, in _transform_core\n",
+      "    data_container = step.handle_transform(data_container, sub_context)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/neuraxle/base.py\", line 807, in handle_transform\n",
+      "    out = self.transform(data_container.data_inputs)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/neuraxle/api/flask.py\", line 63, in transform\n",
+      "    return jsonify(self.encode(data_inputs))\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/flask/json/__init__.py\", line 370, in jsonify\n",
+      "    dumps(data, indent=indent, separators=separators) + \"\\n\",\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/flask/json/__init__.py\", line 211, in dumps\n",
+      "    rv = _json.dumps(obj, **kwargs)\n",
+      "  File \"/usr/lib/python3.6/json/__init__.py\", line 238, in dumps\n",
+      "    **kw).encode(obj)\n",
+      "  File \"/usr/lib/python3.6/json/encoder.py\", line 199, in encode\n",
+      "    chunks = self.iterencode(o, _one_shot=True)\n",
+      "  File \"/usr/lib/python3.6/json/encoder.py\", line 257, in iterencode\n",
+      "    return _iterencode(o, 0)\n",
+      "  File \"/home/alexandre/Documents/LSTM-Human-Activity-Recognition/venv/lib/python3.6/site-packages/flask/json/__init__.py\", line 100, in default\n",
+      "    return _json.JSONEncoder.default(self, o)\n",
+      "  File \"/usr/lib/python3.6/json/encoder.py\", line 180, in default\n",
+      "    o.__class__.__name__)\n",
+      "TypeError: Object of type 'ndarray' is not JSON serializable\n",
```

Comment:
What are those commited error output in the notebook? @alexbrillant

Reply:
it didn't save properly. old errors my bad